server/datastore/mysql/migrations/tables/20240909145426_AddFleetLibraryAppsTable.go

```diff
@@ -0,0 +1,69 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240909145426, Down_20240909145426)
+}
+
+func Up_20240909145426(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE fleet_library_apps (
```

Comment:
Do we care about the `conflicts_with` and `artifacts.zap` fields? They look like they'd be relevant, but I'm not sure if we need to track those for our implementation

Reply:
Are we going to try to automate the creation of an uninstall script from the `zap`s?

Reply:
> Are we going to try to automate the creation of an uninstall script from the zaps?

yeah that's what I was wondering, i can't recall if we talked about that at estimation tho

Reply:
It was mentioned as an option but I don't remember where it landed

Reply:
Yeah my understanding is that we'll create the default install/uninstall scripts from the rest of that metadata, but we don't need to store it otherwise.